### ISSUE TITLE
Custom content in ContextMenu

### ISSUE THREAD

**Issue by HrantAbrahamyan** [dominicstop/react-native-ios-context-menu#1]
After showing the context menu, I just want to display other content.
How to customize content after showing context menu?
Is there any plan to add native custom previewProvider?

![image](https://user-images.githubusercontent.com/36418107/97803215-146d1d00-1c62-11eb-86bc-44b31c94c987.png)

**Comment by dominicstop** [comment]
Hello, i'm currently working on that haha (it's on the [todo list](https://github.com/dominicstop/react-native-ios-context-menu/blob/master/docs/TODO.md), under "_ContextMenuView_")

But for now, what you can do is use `onMenuWillShow` and `onMenuDidShow` to detect if the context menu is visible and change what you render. But the limitation is that the preview size does not change when the context menu is visible, so the new preview content will be clipped if it's bigger (like in the example you shown). 

**TLDR**: It's not a very good workaround since it's limited.

If I finish implementing custom `previewProvider`, I'll let you know and close this issue.

**Comment by HrantAbrahamyan** [comment]
Okay thanks, and one more question, is there any lib for android context menu like ios or do you plan to make this lib cross platform? I will attach examples from android (telegram, instagram).

![photo5258502338405053072](https://user-images.githubusercontent.com/36418107/97815164-a9453a00-1ca5-11eb-8ba7-b1c8bbb67b35.png)
![photo5258502338405053071](https://user-images.githubusercontent.com/36418107/97815168-aba79400-1ca5-11eb-97cb-e688c643a297.png)

**Comment by dominicstop** [comment]
Hello, the library now supports custom previews ✨ (See [Example 11](https://github.com/dominicstop/react-native-ios-context-menu#4111-contextmenuview-simple-example-11) and [Example 12](https://github.com/dominicstop/react-native-ios-context-menu#4112-contextmenuview-simple-example-12)).

Unfortunately, I'm not familiar enough with android development to implement this on android (and as far as i can tell, the android context menu used in instagram/telegram/etc. are custom components?), so this library is iOS only for now 😅

I'll be closing this issue now ☺️

**Comment by HrantAbrahamyan** [comment]
Hey, many thanks ✨
